server/services/prison/PrisonRegisterService.spec.ts

```diff
@@ -154,32 +152,17 @@ describe('Prison Register Service', () => {
     })
   })
 
-  function mockPrisonAddressData() {
```

Comment:
Even though this mock setup was called it was never used at runtime. Dont really understand why because it looks like it should have done, but I took the view that spending too long on it was a waste of time, and simply removed it 🤷‍♂️ 
The downside is that this test is tightly coupled with the contents of `prisonAddressData.json` ... but it was before I came along and removed it .... so .... 🤷‍♂️